### DISCUSSION TITLE
How to hide menu bar on macOS when using BorderlessFullscreen?

### DISCUSSION THREAD

**Discussion by TarekAS** [rust-windowing/winit#3878] [Q&A] [answered]
In some use cases, such as games, we would want the fullscreen mode to take up the entire screen. This includes the menu bar and the dock on macOS. In fact, the [official docs recommends BorderlessFullscreen for games](https://docs.rs/winit/latest/winit/window/struct.Window.html#method.set_fullscreen), yet does not provide options to disable the menu bar.

So, how can I hide the menu bar when using BorderlessFullscreen mode? This is currently a blocker for [bevy](https://github.com/bevyengine/bevy/issues/14783).

**Comment by madsmtm** [comment, marked as answer]
The functionality is not available, opened an issue for it instead https://github.com/rust-windowing/winit/issues/3880
